defaults/main.yml

```diff
@@ -15,6 +15,14 @@ atom_install_dependencies: "true"
 # Install some extra packages specific to our official Vagrant-based
 # development box. See `tasks/devbox.yml` for more details.
 atom_devbox: "no"
+# Use a different AtoM directory for each revision
+atom_revision_directory: "no"
+atom_revision_directory_latest_symlink_dir: "src"
+# Create a symlink from atom_path/private to 
+# atom_path/atom_revision_directory_latest_symlink_dir
+atom_private_symlink: "no"
+#Please, don't change atom_extra_path, it should be ""
+atom_extra_path: ""
 
```

Comment:
I was wondering if there could be a way to avoid defining `atom_extra_path` here (because this is a variable created/modified inside the role, and it is not a variable whose value could be customized by the user)